### DISCUSSION TITLE
White List devices for Playbook Rules

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#3338] [Unsupported Versions] [answered]
Hello all,

After upgrading a standalone to 2.3.30, i receive from playbook various warnings related to:
- Non Interactive PowerShell
- Suspect Svchost Activity
- Wmiprvse Spawning Process

These warnings are all from a Win 2019 multi role server (AD,DNS,DHCP,File)
When I check these windows events, the result is that they are all automatic legitimate events.

I just want to exclude that specific server from those playbook rules, but I can't find any documentation on this anywhere.

Can anyone point me in the right direction?

Regards 
Bart

**Comment by defensivedepth** [comment, marked as answer]
With 2.3.30, the Elasticsearch analyzer we use for Playbook got a bit more accurate, so it makes sense that you are seeing some different alerts.

Playbook Plays are based off of Sigma - you will need to edit the Sigma of the Play to account for local configurations that generate false positives. 

For instance, in one of my SO environments, I am seeing a large amount of  `Non Interactive PowerShell` alerts. Drilling down into the alerts, it appears to be a legitimate execution of `CompatTelRunner.exe`. I went ahead and filtered this out by:

- Copying the Sigma from the Play (found under the Sigma field) and pasting it into the left pane under "Create New Play"
- I then clicked on "Convert" to make sure that it converted correctly
- Then added CompatTelRunner.exe under the filter clause and Converted it again to make sure it worked ([screencap](https://www.screencast.com/t/SShb9dszZ))
- Then I copy and pasted the edited sigma back to the Play under the Sigma field (drop it in between `<pre><code class="yaml">` and `</code></pre> `tags
- Finally, clicked "Submit" - Playbook will take care of the rest.

You can edit the Sigma right there in the Sigma field in the Play, but it is not a YAML editor and sometimes it is easier to edit using a YAML editor.

One caveat - if there is ever an update for that Sigma rule from the Sigma rules repo, your changes will get overritten - we are working on solutions for that & a way to make edits / tuning a bit easier.

Finally - if you are seeing legitimate executions that are not unique to your environment, you might consider submitting a PR to the rule in the Sigma repo. (https://github.com/SigmaHQ/sigma/tree/master/rules)

I will get this written up further and add it to our docs. Please let me know if you have any questions or feedback on this process.

**Reply by sleepingbel** [reply]
Thx 
This is a great help and it is not that difficult to understand the structure in general 

Is there a website where you can find all the filter classes?

There are some other play's that i need to tweek

**Reply by defensivedepth** [reply]
I think this may be what you are looking for?  https://github.com/SigmaHQ/sigma/wiki/Specification#condition
